asset-store/README.md

```diff
@@ -0,0 +1,100 @@
+# Asset Store
+
+## Overview
+
+This example illustrates how to use the [Asset Store](https://kyma-project.io/docs/1.5/components/asset-store/) to store static webpages.
```

Comment:
Can you point to the latest Asset Store version, not explicitly to 1.5? Later we will switch this link to Rafter.